common/src/main/kotlin/org/jetbrains/teamcity/vault/VaultFeatureSettings.kt

```diff
@@ -15,11 +15,16 @@
  */
 package org.jetbrains.teamcity.vault
 
-data class VaultFeatureSettings(val url: String, val endpoint: String, val roleId: String, val secretId: String) {
+data class VaultFeatureSettings(val parameterPrefix: String, val url: String, val endpoint: String, val roleId: String, val secretId: String) {
```

Comment:
Probably could be simplified to `prefix`.
Though I've idea to restrict possible prefixes like: `vault:`,`vault-X:`, `vault-Y:` and so. It would be easier to distinguish such parameters by human.

Reply:
Then maybe I should rename the prefix to namespace

Reply:
I couldn't find the "reformat changed code" option (i am a bit new to intellij). I did a regular interactive rebase and reformatted the parts that were in conflict with "reformat code"